code/_globalvars/lists/client.dm

```diff
@@ -1,2 +1,3 @@
 GLOBAL_LIST_EMPTY(keybindings_by_name)
 GLOBAL_LIST_EMPTY(keybindings_by_name_to_key)
+GLOBAL_LIST_EMPTY(total_votes_to_leave)
```

Comment:
This GLOB should be in SS.

Reply:
There isn't an SS file for globals.

Reply:
Alright, I made a subsystem.dm file just for this.

Reply:
Bacon resolved this now.